### DISCUSSION TITLE
CI Runners with GPUs

### DISCUSSION THREAD

**Discussion by ssinad** [ivadomed/ivadomed#800] [Ideas]
This may have a low priority, but it is something to have at the back of our minds.  
It would be nice to have runners in addition to those provided by GitHub that have GPU capability.
https://docs.github.com/en/actions/hosting-your-own-runners/adding-self-hosted-runners

**Comment by ssinad** [comment]
Do we have access to GPU-enabled servers at Neuropoly?

**Reply by jcohenadad** [reply]
yes we do (`rosenberg`, `romane` and `bireli`)

**Comment by dyt811** [comment]
@ssinad, so we recently tested GitLab based privately hosted GPU runners to help run GPU tests. Still highly experimental but definitely works as a proof of concept (and already discovered several bugs). Ping me on Slack and I can tell you more detail about this.
